coalib/settings/Setting.py

```diff
@@ -89,7 +113,7 @@ def __init__(self,
                                            last part will be stripped of. If
                                            you want to specify a directory as
                                            origin be sure to end it with a
-                                           directory seperator.
+                                           directory separator.
```

Comment:
For future reference, this should be in an own commit.
Im letting it go because i just want this merged soon ... (In case you are redo-ing this, please dont fix this here)